openapi/components/schemas/Coupon/CouponRestrictions/restrict-to-invoice-items-with-plans.yaml

```
@@ -0,0 +1,13 @@
+description: Apply restrictions to invoice items with the specified Plan Ids.
```

```suggestion
description: Restricts a coupon redemption to invoice items with the specified Plan Ids only.
```
It's just a suggestion, because I think the term `restriction` is more like domain name of the enum of possible restrictions types